packages/address/src/loader.ts

```diff
@@ -94,3 +98,20 @@ export function toSupportedLocale(locale: string) {
     return DEFAULT_LOCALE;
   }
 }
+
+type AsyncFunc = (...args: any[]) => Promise<any>;
+interface Cache {
+  [key: string]: Promise<any>;
+}
+
+function memoizeAsync(asyncFunction: AsyncFunc) {
+  const cache: Cache = {};
+
+  return (..._arguments: any[]) => {
```

Comment:
Why do you need to prefix `arguments` with `_`. The param is used in your function.

Reply:
I did that because `arguments` was a used keywords and `args` was already taken.
But I've just renamed it 👍